sdks/python/container/boot.go

```diff
@@ -170,6 +176,34 @@ func main() {
 	log.Fatalf("Python exited: %v", execx.Execute("python", args...))
 }
 
+// setup wheel specs according to installed python version
+func setupAcceptableWheelSpecs() error {
+	cmd := exec.Command("python", "-V")
+	stdoutStderr, err := cmd.CombinedOutput()
+	if err != nil {
+		return err
+	}
+	re := regexp.MustCompile(`Python (\d)\.(\d).*`)
+	pyVersions := re.FindStringSubmatch(string(stdoutStderr[:]))
+	if len(pyVersions) != 3 {
+		return fmt.Errorf("cannot get parse Python version from %s", stdoutStderr)
+	}
+	pyVersion := fmt.Sprintf("%s%s", pyVersions[1], pyVersions[2])
+	if pyVersion == "27" {
+		acceptableWhlSpecs = append(acceptableWhlSpecs, "cp27-cp27mu-manylinux1_x86_64.whl")
```

Comment:
Can we move this into the `case` as well?  Thank you!

Reply:
Updated, PTAL.